### ISSUE TITLE
Set expectations about large datasets

### ISSUE THREAD

**Issue by mekline** [psych-ds/psych-DS#41]
# TLDR

At some point, datasets become long enough that they cause high latency or even timeout on the webapp implementation. 

We don't need to provide super duper duper duper performance (very large datasets probably want CLI validation for other reasons, maybe?) but we should give people an indication of what to expect. 

# Details

Once low-hanging fruit speed improvements are done, run some large datasets and add (somewhere on website, probably documentation?) some information like "Datasets with files larger than X may take Y amount of time", "Datasets with more than X files may take Z amount of time."
